<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBackfill</class>
 <widget class="QWidget" name="FormBackfill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Состояние бункеров на</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="displayFormat">
        <string>dd.MM.yy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdStat">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdStatTime">
       <property name="text">
        <string>Обновить на текущее время</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Цех</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DbComboBox" name="comboBoxCex"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="TableView" name="tableViewBunk">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="TableView" name="tableViewPart">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Засыпка бункеров компонентами</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>С</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DbDateEdit" name="dateEditBeg"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>По</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DbDateEdit" name="dateEditEnd"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpd">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DbViewer" name="tableViewLoad"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>DbDateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>DbComboBox</class>
   <extends>QComboBox</extends>
   <header>db/dbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
